staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -38,10 +41,15 @@ type AuditLogOptions struct {
 	MaxSize    int
 
 	PolicyFile string
+
+	WebhookFile string
```

Comment:
Actually, can we restructure this into `AuditOptions`, `AuditLogOptions` and `AuditWebhookOptions` ?